src/server/db/index.js

```diff
@@ -25,5 +25,8 @@ if (!config) {
 
 const connection = knex(config);
 const db = new Bookshelf(connection);
+db.plugin('registry');
 
 export default db;
+
+require('./models');
```

Comment:
What is this `require` supposed to do? It's not assigned anywhere, it doesn't seem to be exported...

Why not `import`?

I suppose it's some kind of a convention for bookshelf, but it looks quite confusing when looking at the code.